odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/mapper/ToFhirMapping.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.odoo.openmrs.mapper;
+
+import org.hl7.fhir.r4.model.Resource;
+
+/**
+ * An Interface for mapping from FHIR Resources to Odoo Documents
+ *
+ * @param <F> FHIR Resource
+ * @param <E> Odoo Document
+ */
+public interface ToFhirMapping<F extends Resource, E> {
```

Comment:
This is not intuitive enough, since it says ToFhirMapping, then the type arguments need to reflect that i.e. the Fhir type should be the second in the type parameters.

Reply:
Yes I agree. I just want to keep things consistent. This is how it is being done in [eip-erpnext-openmrs](https://github.com/ozone-his/eip-erpnext-openmrs/blob/main/erpnext-fhir-mapper/src/main/java/com/ozonehis/eip/mappers/ToERPNextMapping.java#L18)

cc: @corneliouzbett

Reply:
Make sense to me, @VaishSiddharth I think you should make the change. Will make the change to `eip-erpnext-openmrs`